src/alr/alr-commands-test-runner.ads

```diff
@@ -0,0 +1,36 @@
+with AAA.Strings;
```

Comment:
What's the difference between this command and `alr test`?

Reply:
This commands only runs the test runner in the current directory, without reading the test runner config in `alire.toml` and switching to any subfolder. It _could_ be useful if someone wants to run the test runner in a shell script.